CHANGELOG.md

```diff
@@ -36,7 +36,7 @@
   representation of the urn. Instead of using `URNEntityType.name()`
   `URNEntityType.getEntityValue()` should be used to support UGCPosts.
 
-## 1.0.3 (Work in progress)
+## 1.0.3
```

Comment:
We never actually had a 'non snapshot' release of 1.0.3 yet. I'd propose doing that after this PR has been merged, see 'Releases' section - https://github.com/ebx/ebx-linkedin-sdk